.travis.yml

```diff
@@ -30,6 +30,8 @@ matrix:
     env: CONDA_ENV=py35-dask-dev
   - python: 3.5
     env: CONDA_ENV=py35-pandas-dev
+  - python: 3.5
+    env: CONDA_ENV=py35-mpl2
```

Comment:
Yes. I also renamed the environment, which is set to always use the release candidates when available. I don't know if it makes sense to keep this test environment once mpl v2 is released...